### PR TITLE
v7.3.2

### PR DESCRIPTION
**Pull request by JCQuintas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,70 @@
...
+
+#### `@mui/x-charts@7.3.2`
+
+- [charts] Add an overlay for "no data" or "loading" states (#12817) @alexfauquette
+- [charts] Fix typos in documentation, translations and errors (#12941) @JCQuintas
+- [charts] Fix `prop.slots` and `prop.slotProps` not passed to `<ChartsTooltip />` (#12939) @JCQuintas
+
+### Tree View
+
+#### `@mui/x-tree-view@7.3.2`
```
Comment (JCQuintas) [inline CHANGELOG.md:55]: Do we bump when there are no changes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,66 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.3.2
+
+_May 2, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add "no data" and "loading" states to charts, allowing users to create [custom visualizations for each state](https://mui.com/x/react-charts/styling/#overlay)
+- 📝 Fixed a typo in an undocumented [x-charts gradient id](https://github.com/mui/mui-x/pull/12941/files/4bf4bffbc2799a01a96bc7458a17318cf41c1722#diff-26c31cc69d6f51110f89e339578ef9b3d4a3551f79077fff73f7babb81c5099f) that could have unintended side effects
```
Comment (LukasTy) [inline CHANGELOG.md:13]: Nitpick: Is it worth adding such a thing into highlights? 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,66 @@
...
+- 🌍 Improve Catalan (he-IL) and Spanish (hu-HU) locales on the Date and Time Pickers
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.3.2`
+
+- [DataGrid] Derive `formattedValue` from the edit value when passing to `renderEditCell` (#12870) @cherniavskii
+
```
Comment (MBilalShafi) [inline CHANGELOG.md:23]:
```suggestion
- [DataGrid] Fix rows not being recomputed on `props.rowCount` change (#12833) @MBilalShafi

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,66 @@
...
+#### `@mui/x-data-grid-pro@7.3.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@7.3.2`.
+
+#### `@mui/x-data-grid-premium@7.3.2` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.3.2`.
```
Comment (MBilalShafi) [inline CHANGELOG.md:30]:
```suggestion
Same changes as in `@mui/x-data-grid-pro@7.3.2`, plus:

- [DataGridPremium] Fix print export not working with row grouping (#12957) @MBilalShafi

```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,66 @@
...
+
+#### `@mui/x-charts@7.3.2`
+
+- [charts] Add an overlay for "no data" or "loading" states (#12817) @alexfauquette
+- [charts] Fix typos in documentation, translations and errors (#12941) @JCQuintas
+- [charts] Fix `prop.slots` and `prop.slotProps` not passed to `<ChartsTooltip />` (#12939) @JCQuintas
+
+### Docs
+
+- [docs] Polish references to the plans (#12922) @oliviertassinari
```
Comment (MBilalShafi) [inline CHANGELOG.md:55]:
```suggestion
- [docs] Improve Data Grid migration guide (#12969) @MBilalShafi
- [docs] Polish references to the plans (#12922) @oliviertassinari
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,66 @@
...
 
+## 7.3.2
+
+_May 2, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add "no data" and "loading" states to charts, allowing users to create [custom visualizations for each state](https://mui.com/x/react-charts/styling/#overlay)
+- 📝 Fixed a typo in an undocumented [x-charts gradient id](https://github.com/mui/mui-x/pull/12941/files/4bf4bffbc2799a01a96bc7458a17318cf41c1722#diff-26c31cc69d6f51110f89e339578ef9b3d4a3551f79077fff73f7babb81c5099f) that could have unintended side effects
+- 🌍 Improve Catalan (he-IL) and Spanish (hu-HU) locales on the Date and Time Pickers
```
Comment (MBilalShafi) [inline CHANGELOG.md]:
```suggestion
- 🌍 Improve Hebrew (he-IL) and Hungarian (hu-HU) locales on the Date and Time Pickers
```

File: CHANGELOG.md (hunk 1 above)
Comment (LukasTy) [reply CHANGELOG.md]: Lately we've been releasing whatever lerna detects, that sometimes means that packages, which only received dependency bumps would also be picked up.
This decision has not been finalized yet, but we did release such packages with the last few releases.
Personally, I lean more towards not releasing what has not changed, but that's just my opinion with "hyper-optimisation" in mind. 😆

File: CHANGELOG.md (hunk 6 above)
Comment (LukasTy) [inline CHANGELOG.md:14]:
```suggestion
- 🌍 Improve Hebrew (he-IL) and Hungarian (hu-HU) locales on the Date and Time Pickers
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,66 @@
...
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.3.2`
+
```
Comment (MBilalShafi) [inline CHANGELOG.md:21]:
```suggestion

[l10n] Improve Slovak (sk-SK) locale (#12949) @stefikp
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,66 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.3.2
+
+_May 2, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
```
Comment (MBilalShafi) [inline CHANGELOG.md:10]:
```suggestion
We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
```

**Review by MBilalShafi — APPROVED** [review]
Merged a few PRs

File: CHANGELOG.md [hunk 9]
```diff
@@ -3,6 +3,66 @@
...
+
+- 🎁 Add "no data" and "loading" states to charts, allowing users to create [custom visualizations for each state](https://mui.com/x/react-charts/styling/#overlay)
+- 📝 Fixed a typo in an undocumented [x-charts gradient id](https://github.com/mui/mui-x/pull/12941/files/4bf4bffbc2799a01a96bc7458a17318cf41c1722#diff-26c31cc69d6f51110f89e339578ef9b3d4a3551f79077fff73f7babb81c5099f) that could have unintended side effects
+- 🌍 Improve Hebrew (he-IL) and Hungarian (hu-HU) locales on the Date and Time Pickers
+- 🐞 Bugfixes
```
Comment (MBilalShafi) [inline CHANGELOG.md:14]:
```suggestion
- 🌍 Improve Slovak (sk-SK) locale on the Data Grid
- 🐞 Bugfixes
```

File: CHANGELOG.md (hunk 2 above)
Comment (JCQuintas) [reply CHANGELOG.md]: I was unsure where to put it, moved it to just below the commit

File: CHANGELOG.md [hunk 10]
```diff
@@ -48,6 +47,8 @@ Same changes as in `@mui/x-date-pickers@7.3.2`.
 
 - [charts] Add an overlay for "no data" or "loading" states (#12817) @alexfauquette
 - [charts] Fix typos in documentation, translations and errors (#12941) @JCQuintas
+  This affects an undocumented [x-charts gradient id](https://github.com/mui/mui-x/pull/12941/files/4bf4bffbc2799a01a96bc7458a17318cf41c1722#diff-26c31cc69d6f51110f89e339578ef9b3d4a3551f79077fff73f7babb81c5099f) that could have unintended side effects
```
Comment (MBilalShafi) [inline CHANGELOG.md:54]: Should we add this under the breaking changes section?

File: CHANGELOG.md (hunk 1 above)
Comment (JCQuintas) [reply CHANGELOG.md]: I removed it for now as @michelengelen suggested. I don't have any preference, but would like to add our decision to the readme file so it is documented for the next new joiner 😅

File: CHANGELOG.md [hunk 11]
```diff
@@ -3,6 +3,74 @@
...
+- 🌍 Improve Hebrew (he-IL) and Hungarian (hu-HU) locales on the Date and Time Pickers
+- 🌍 Improve Slovak (sk-SK) locale on the Data Grid
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.3.2`
+
+[l10n] Improve Slovak (sk-SK) locale (#12949) @stefikp
```
Comment (MBilalShafi) [inline CHANGELOG.md:22]:
```suggestion
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -3,6 +3,74 @@
...
+- [DataGrid] Derive `formattedValue` from the edit value when passing to `renderEditCell` (#12870) @cherniavskii
+- [DataGrid] Fix rows not being recomputed on `props.rowCount` change (#12833) @MBilalShafi
+
```
Comment (MBilalShafi) [inline CHANGELOG.md:25]:
```suggestion
- [l10n] Improve Slovak (sk-SK) locale (#12949) @stefikp

```

File: CHANGELOG.md (hunk 2 above)
Comment (LukasTy) [reply CHANGELOG.md]: Oh, sorry, I didn't check what is all it about. 🙈 
Yeah, in that case, I think that mentioning it in highlights makes much more sense...
Sorry for the confusion. 🤷 
I'd maybe rephrase it to be more specific, targetting the actual problem - change in the id of an element instead of saying "unintended side effects", which sounds more scary to me, when reading about it. 🙈

**Review by MBilalShafi — COMMENTED** [review]
Updating the order and fixing the bullet point.

File: CHANGELOG.md (hunk 10 above)
Comment (JCQuintas) [reply CHANGELOG.md:56]: Technically a breaking change yes, but it is also very small. @LukasTy suggested just adding it to the changelog. Do you think this should be in a `### Breaking change` section?

File: CHANGELOG.md (hunk 1 above)
Comment (LukasTy) [reply CHANGELOG.md]: I'm fine with it.
Totally agree that it should be documented once we land on a decision. 👌

File: CHANGELOG.md (hunk 10 above)
Comment (LukasTy) [reply CHANGELOG.md:56]: IMHO, it's such a small BC that flagging it as one might be a bit counter-intuitive and cause more confusion than necessary. 🙈

Comment (MBilalShafi) [reply CHANGELOG.md:56]: Go ahead with what you feel better, IMHO, putting in the breaking changes section would make more sense, as we used to do the same for all other BCs.

File: CHANGELOG.md (hunk 2 above)
Comment (JCQuintas) [reply CHANGELOG.md]: done

**Review by LukasTy — APPROVED** [review]
Nice. 👍 
LGTM! 🚀

File: CHANGELOG.md [hunk 13]
```diff
@@ -3,6 +3,76 @@
...
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.3.2`
+
+- [DataGrid] Derive `formattedValue` from the edit value when passing to `renderEditCell` (#12870) @cherniavskii
```
Comment (MBilalShafi) [inline CHANGELOG.md:26]: Could we add this one too please?
```suggestion
- [DataGrid] Allow to change reset text in columns management panel (#12972) @MBilalShafi
- [DataGrid] Derive `formattedValue` from the edit value when passing to `renderEditCell` (#12870) @cherniavskii
```

**Review by noraleonte — APPROVED** [review]
LGTM 🎉 
Congrats on your first release PR! 🍰

**Review by michelengelen — APPROVED** [review]
LGTM ... :shipit: 
Congrats on your first release PR! 🎉
